src/Microsoft.AspNetCore.Sockets/ConnectionMetadataNames.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Microsoft.AspNetCore.Sockets
+{
+    public static class ConnectionMetadataNames
+    {
+        public static readonly string Format = nameof(Format);
```

Comment:
IMO, no, ordinal is quicker and these are tokens that are designed to be used from this static list.